script/NestedSignFromJson.s.sol

```diff
@@ -36,25 +36,4 @@ contract NestedSignFromJson is NestedMultisigBuilder, JsonTxBuilderBase {
     function _ownerSafe() internal view override returns (address) {
         return vm.envAddress("OWNER_SAFE");
     }
-
-    function _postCheck(Vm.AccountAccess[] memory accesses, SimulationPayload memory simPayload)
-        internal
-        virtual
-        override
-    {
-        if (msg.sig == this.approveJson.selector) {
-            console.log("Skipping assertions on the approval call");
-            return;
-        }
```

Comment:
the `NestedMultisigBuilder` no longer calls `_postCheck` from the approval call, so this can be cleaned up

Reply:
This makes sense, I like the new `_postCheck`, `_postSign`, `_postApprove`, `_postRun` split in `NestedMultisigBuilder`